src/components/auth/SignInScreen.js

```diff
@@ -90,11 +91,10 @@ export default class SignInScreen extends React.Component {
                   </Item>
                   {/* Password */}
                   <Item style={styles.itemStyle}>
-                    <Ionicons style={styles.iconStyle} name="ios-lock" />
                     <Input
                       style={styles.input}
-                      placeholder={i18n.t('Password')}
-                      placeholderTextColor={COLORS.lightblue}
+                      placeholder={"Senha"}
```

Comment:
## refer to: [!Comment Above](https://github.com/wang1kevin1/InstitutoTerra/pull/43/files#r451781286)